### PR TITLE
First integration documentation: Datadog

### LINKED ISSUE #773: DataDog integration docs, metrics and logs
# What should we add to our content collections?

DataDog is our most-used integration, migrate/update the content around this integration. There are a variety of existing documents that should come together into a few more coherent content types:

* https://help.aiven.io/en/articles/2933115-remote-syslog-integration (this also covers other rsyslog targets, so the work in this PR won't mean we need to redirect from that article yet)
* https://help.aiven.io/en/articles/1759208-getting-started-with-datadog shows how to set up the metrics integration
* https://help.aiven.io/en/articles/3359501-configuring-datadog-metrics-with-aiven-for-apache-kafka has some extra kafka-specific tips for configuration
* https://help.aiven.io/en/articles/5088895-missing-kafka-datadog-metrics more kafka-specific metrics tips, this time about things that we do not expose
* https://help.aiven.io/en/articles/5223432-missing-postgresql-datadog-metrics mentions some limitation of the postgres integration, this is also already listed as an issue in #304 
* https://help.aiven.io/en/articles/5384590-datadog-integration-in-aiven-services is a nice intro that we should definitely migrate or improve on
* https://help.aiven.io/en/articles/5372887-adding-custom-tags-to-your-datadog-integration-in-the-aiven-web-console and https://help.aiven.io/en/articles/5384618-adding-custom-tags-to-your-datadog-integration-with-the-aiven-cli both deal with adding tags to your metrics; these could be linked to for now, and migrated later.

# Is this something you would work on yourself?

[x] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by lornajane**
# What changed, and why it matters

The integration with DataDog is in good shape but it handles metrics and logs separately, and they aren't always intuitive to set up. This adds a new "integrations" section, and some starter docs for our DataDog integrations. Fixes #773

### CODE REVIEW

File: docs/integrations/howto/datadog-logs.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Send logs to DataDog
+====================
+
+This article will show you how to use the Aiven Rsyslog integration to send the logs from your Aiven services to DataDog.
+
+You will need:
+
+* A DataDog account.
+
+* The region to use, US or EU
```
Comment (anelook) [inline docs/integrations/howto/datadog-logs.rst:10]: While other two points are clear, "The region to use, US or EU" leaves me wonder what exactly I'll need, is it that we support only these two regions? are they what we need in DataDog or on Aiven's side?

File: docs/integrations/howto/datadog-metrics.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+=======================
+
+This guide will show you how to send metrics from your Aiven service to your DataDog account.
+
+Prepare your DataDog account
+----------------------------
+
+You will need a DataDog account and the following details handy:
+
+* Which region you want to use, US or EU
```
Comment (anelook) [inline docs/integrations/howto/datadog-metrics.rst:11]: Similar note as in the other article

File: docs/integrations/howto/datadog-metrics.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+* An API key for this integration. Generate an API key by visiting **Organization settings** under your account menu, and then choose **API Keys**. The **New Key** button will give you an API key; you should copy this as you will need it shortly.
```
Comment (anelook) [inline docs/integrations/howto/datadog-metrics.rst:13]: As in previous article, let's be specific that we're talking about "A DataDog API key."

**Review by anelook — CHANGES_REQUESTED** [review]
I carefully read through the material, however, I didn't test these steps with DataDog account.
It is a smooth and easy read, only couple of places confused me, I left specific points there.

Should I set up DataDog account and run the actual steps to verify, or can we rely that it is already tested?

File: docs/integrations/howto/datadog-logs.rst (hunk 1 above)
Comment (lornajane) [reply docs/integrations/howto/datadog-logs.rst:10]: With DataDog, you have to actually register in one region or the other. Let me try to make that clearer

Comment (lornajane) [reply docs/integrations/howto/datadog-logs.rst]: Updated, hopefully makes more sense.

File: docs/integrations/howto/datadog-metrics.rst (hunk 3 above)
Comment (lornajane) [reply docs/integrations/howto/datadog-metrics.rst]: Good catch! Fixed.

**Comment by lornajane** [comment]
No need to re-test, I wrote the docs and then (after I'd slept!) used them to set up the demo for the Let's Talk Aiven Tech session last week so they should be pretty much sorted.

**Review by anelook — APPROVED** [review]
Looks great!! Thank you, Lorna!

**Pull request merged**